hasher-matcher-actioner/reference_omm_configs/development_omm_config.py

```diff
@@ -56,3 +56,11 @@
 
 # Debugging stuff
 # SQLALCHEMY_ENGINE_LOG_LEVEL = logging.INFO
+
+# URL validation configuration
+# ALLOWED_HOSTNAMES = {
```

Comment:
Perhaps we should have a toggle for `ALLOW_REMOTE_FILES` (and maybe `ALLOW_FILE_UPLOADS`) to completely disable

Reply:
Not sure why we would need to disable both. Disabling URLs. Also this can be something enabled/disabled by the consumer of the API. If no hashing needs to be happening we already have disabled the whole hashing path as part of the role for the container.